pkgs/applications/misc/timewarrior/default.nix

```diff
@@ -1,18 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, cmake }:
+{ lib, stdenv, fetchFromGitHub, cmake, asciidoctor }:
 
 stdenv.mkDerivation rec {
   pname = "timewarrior";
-  version = "1.4.2";
+  version = "1.4.3";
 
   src = fetchFromGitHub {
     owner = "GothenburgBitFactory";
     repo = "timewarrior";
     rev = "v${version}";
-    sha256 = "0qvhpva0hmhybn0c2aajndw5vnxar1jw4pjjajd2k2cr6vax29dw";
+    sha256 = "00ydikzmxym5jhv6w1ii12a6zw5ighddbzxsw03xg8yabzzfnvzw";
     fetchSubmodules = true;
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake asciidoctor ];
```

Comment:
> A notable change is the transition from roff man pages to asciidoc generated man pages.

Why are you using asciidoctor then?

Reply:
The man pages now need asciidoctor to generate them.

https://github.com/GothenburgBitFactory/timewarrior/blob/35dc22c8efa56735225b32da38b47d86fda40759/doc/CMakeLists.txt#L19

Reply:
I was somehow confused about ascidoc and ascidoctor